zipkin-lens/webpack.dev.config.js

```diff
@@ -59,13 +59,16 @@ module.exports = {
       {
         test: /\.svg$/,
         use: [
-          {
-            loader: 'babel-loader',
-          },
+          { loader: 'babel-loader' },
           {
             loader: 'react-svg-loader',
             options: {
               jsx: true,
+              svgo: {
+                plugins: [
+                  { removeViewBox: false },
+                ],
+              },
```

Comment:
In `react-svg-loader@3.0.3`, `viewBox` attribute is removed by default.
If there is not `viewBox`, SVG icon will no be showed now.
I need to add `svgo` options like this for avoiding to remove `viewBox` attribute.

* react-svg-loader reference → https://github.com/boopathi/react-svg-loader/tree/master/packages/react-svg-loader#svgo-options
* svgo reference → https://github.com/svg/svgo#what-it-can-do

Reply:
very good notes